src/boot/AppEventHandlers.js

```diff
@@ -131,7 +125,12 @@ class AppEventHandlers extends PureComponent<Props> {
   }
 
   render() {
-    return <View style={styles.wrapper}>{this.props.children}</View>;
+    return (
+      <>
```

Comment:
Huh interesting! What's this syntax do? Looks potentially handy elsewhere too.

(I searched for `<>` in the two most obvious pages of JSX docs -- https://reactjs.org/docs/introducing-jsx.html and https://reactjs.org/docs/jsx-in-depth.html -- with no luck.)

Reply:
It's just a grouping.
* https://reactjs.org/docs/fragments.html
* https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
TIL, thanks!

... I'll move to [a chat thread](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/react.20fragments/near/800635).